app/subscriber/src/components/content-list/ContentList.tsx

```diff
@@ -62,13 +62,20 @@ export const ContentList: React.FC<IContentListProps> = ({ content, setSelected,
                   selected.some((selectedItem) => selectedItem.id === item.id) ? 'checked' : ''
                 }`}
                 key={item.id}
-                onClick={() => navigate(`/view/${item.id}`)}
+                onClick={(e) => {
```

Comment:
Does anyone know why `e.stopPropegation` is not working on line 78? Spent about an hour debugging. Don't like this approach.

Reply:
Did you try `preventDefault()`?

Reply:
Same thing :( - not sure why...